packages/internal/generated-clients/Makefile

```diff
@@ -42,15 +46,34 @@ generate-mr-api-client:
     -o /app/src/multi-rollup \
     -c /app/config/mr.config.json
 
-# When running this command, ensure files not relevant to blockchain data are manually removed
+# -------------------------------------------------
+# Seperate clients
+# -------------------------------------------------
+
+.PHONY: generate-blockchain-data
+generate-blockchain-data: get-blockchain-data-openapi generate-blockchain-data-client
+
+.PHONY: get-blockchain-data-openapi
+get-blockchain-data-openapi:
+	rm -f src/blockchain-data-openapi.json && touch src/blockchain-data-openapi.json && \
```

Comment:
Could you use rimraf here instead of rm -f, as below? rimraf will let this work on windows as well

Reply:
Done 👍